### DISCUSSION TITLE
How to interpret punctual receiver intensity

### DISCUSSION THREAD

**Discussion by ToonC08** [Ifsttar/I-Simpa#352] [General]
Hi guys,
I am currently working on a school project and i am trying to visualize active noise control in i simpa. In one simulation i put two speakers next to eachother with one playing a normal 200hz tone and the other one the inverted wave of 200hz. In the second simulation i only let one of the two speakers play the tone. I am trying to visualize the results of the receiver in order to see if i simpa is able to calculate these kind of noise cancelling arrays. How do i read the values i simpa gives me when i choose punctual receiver intensity as shown below? : 
![image](https://user-images.githubusercontent.com/98044358/222418322-7fc65e7b-6780-493a-a1c4-55c5bb00cf01.png)

**Comment by ToonC08** [comment]
Also these d50 cuts show a difference in the length of the waves so there should be results, i only do not know how to obtain them.
![image](https://user-images.githubusercontent.com/98044358/222418812-e8c7fa89-28c3-462c-82fb-1a377e1f7b2d.png)
![image](https://user-images.githubusercontent.com/98044358/222418841-742869ba-fabb-4a49-be75-7c2b268e5283.png)

**Comment by nicolas-f** [comment]
Hi,

SPPS computation core does not simulate interaction between sound waves:

https://i-simpa-wiki.readthedocs.io/fr/latest/code_SPPS_principle.html

You have to use finite element method in order to do this kind of simulations, such as in the following project:

https://acousto.sourceforge.net/

Best regards,
